core/arch/arm/table_t32_coproc.c

```diff
@@ -504,7 +504,7 @@ const instr_info_t T32_ext_opc4fpA[][3] = {
     {INVALID,     0xee200a10, "(bad)",  xx, xx, xx, xx, xx, no, x, NA},
     {OP_vnmul_f32,0xee200a40, "vnmul.f32", WBd, xx, WAd, WCd, xx, vfp, x, END_LIST},
   }, { /* 3 */
-    {OP_vadd_f32, 0xee300a00, "vadd.f32", WBd, xx, WAd, VCd, xx, vfp, x, END_LIST},
+    {OP_vadd_f32, 0xee300a00, "vadd.f32", WBd, xx, WAd, WCd, xx, vfp, x, END_LIST},
```

Comment:
I believe the entries in table_a32_pred.c are also incorrect.  Could you fix those as well?

```
$ git grep 'W.*xx.*W.* V'
core/arch/arm/table_a32_pred.c:    {OP_vadd_f32, 0x0e300a00, "vadd.f32", WBd, xx, WAd, VCd, xx, pred|vfp, x, END_LIST},
core/arch/arm/table_a32_pred.c:    {OP_vadd_f32, 0x0e700a00, "vadd.f32", WBd, xx, WAd, VCd, xx, pred|vfp, x, DUP_ENTRY},
core/arch/arm/table_t32_coproc.c:    {OP_vadd_f32, 0xee300a00, "vadd.f32", WBd, xx, WAd, VCd, xx, vfp, x, END_LIST},
core/arch/arm/table_t32_coproc.c:    {OP_vadd_f32, 0xee700a00, "vadd.f32", WBd, xx, WAd, VCd, xx, vfp, x, DUP_ENTRY},
```

Reply:
Done! I've pushed another commit.